kernel/nano_core/linker_higher_half-aarch64.ld

```diff
@@ -0,0 +1,103 @@
+ENTRY(_start)
+OUTPUT_FORMAT(elf64-littleaarch64)
+
+/* the last 2GiB of mappable memory */
+KERNEL_OFFSET = 0xFFFF80000000;
```

Comment:
```suggestion
/*
 * The upper-most 2GiB of accessible virtual addresses on aarch64,
 * because Theseus uses 48-bit virtual addresses with an ASID of 0.
 */
KERNEL_OFFSET = 0x0000_FFFF_8000_0000;
```

Reply:
underscores aren't allowed but apart from that, np